pkg/metrics/status.go

```diff
@@ -15,19 +15,13 @@
 package metrics
 
 import (
-	"time"
-
 	clientPkg "github.com/cilium/cilium/pkg/client"
 	healthClientPkg "github.com/cilium/cilium/pkg/health/client"
 
 	"github.com/prometheus/client_golang/prometheus"
 	log "github.com/sirupsen/logrus"
 )
 
-const (
-	updateLatencyMetricsInterval = 30 * time.Second
```

Comment:
:thinking:

Reply:
Seems that the const has not been used since the beginning: https://github.com/cilium/cilium/pull/4437/ (also `git log -S updateLatencyMetricsInterval`).